pyproject.toml

```diff
@@ -0,0 +1,61 @@
+[build-system]
+requires = [
+    "hatchling>=0.11.2",
+    "setuptools<61",
+]
+build-backend = "hatchling.build"
+
+[project]
+name = "test-checks-downloader"
+description = "The test Checks Downloader"
+readme = "README.md"
+license = { text = "BSD-3-Clause" }
+keywords = [
+    "test",
+    "test agent",
+    "test check",
+]
+authors = [
+    { name = "test", email = "test@test.com" },
+]
+classifiers = [
+    "Private :: Do Not Upload",
+    "Development Status :: 5 - Production/Stable",
+    "Intended Audience :: Developers",
+    "Intended Audience :: System Administrators",
+    "License :: OSI Approved :: BSD License",
+    "Programming Language :: Python :: 2.7",
+    "Programming Language :: Python :: 3.8",
+    "Topic :: System :: Monitoring",
+]
+dynamic = [
+    "version",
+]
+
+dependencies = [
+    "Werkzeug==2.3.6",
+    "Jinja2<3.1.2",
+    "itsdangerous>2.1.2",
+    "click>=8.1.3",
+    "blinker<=1.6.2",
+    "importlib-metadata>=3.6.0; python_version < '3.10'",
+    "importlib-metadata==3.8.4; python_version >= '3.10'",
+    "piccolo<1.1.1",
+]
+
+[project.optional-dependencies]
+deps = [
+    "in-toto==1.0.1",
```

Comment:
:exclamation:**Cycode**: Security vulnerability found in newly introduced dependency. 

||| 
 |---|---|
|**Severity**|Medium|
|**Issue**|in-toto vulnerable to Configuration Read From Local Directory: CVE-2023-32076|
|**Ecosystem**|PyPI|
|**Dependency**|`in-toto`|
|**Dependency Paths**|`in-toto 1.0.1`|
|**Direct Dependency**|Yes|
|**Upgrade**|`2.0.0`|

 
 ### Impact

The in-toto configuration is read from various directories and allows users to configure the behavior of the framework. The files are from directories following the XDG base directory specification [1]. Among the files read is `.in_totorc` which is a hidden file in the directory in which in-toto is run. If an attacker controls the inputs to a supply chain step, they can mask their activities by also passing in an `.in_totorc` file that includes the necessary exclude patterns and settings.

RC files are widely used in other systems [2] and security issues have been discovered in their implementations as well [3]. We found in our conversations with in-toto adopters that `in_totorc` is not their preferred way to configure in-toto. As none of the options supported in `in_totorc` is unique, and can be set elsewhere using API parameters or CLI arguments, we decided to drop support for `in_totorc`.

### Other Recommendations

Sandbox functionary code as recommended in https://github.com/in-toto/docs/security/advisories/GHSA-p86f-xmg6-9q4x.

### References

[1] https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html
[2] https://spec.editorconfig.org/
[3] https://github.blog/2022-04-12-git-security-vulnerability-announced/

 <!-- ab059e91-0f03-488f-8375-bcb797a4968d -->